pom.xml

```diff
@@ -424,6 +424,13 @@
             <version>31.0.1-jre</version>
         </dependency>
 
+        <!-- https://github.com/failsafe-lib/failsafe-->
```

Comment:
Do we need it for ingest SDK to trigger white source?

Reply:
This PR triggered whitesource with no vuln. so i think we are good to go..

Reply:
(This dependency is not for triggerring whitesource, this is for retries, fallback and circuitbreaker)
Checkout: https://github.com/failsafe-lib/failsafe